### PR TITLE
[aurora-postgres] fix tflint and markdownlint

### PR DESCRIPTION
**Pull request by kevcube**
## what

- fix tflint and markdownlint complaints

## why

- we want to use this component and instead of just fixing in our repo, I upstream

### CODE REVIEW

File: modules/aurora-postgres/variables.tf
Comment (kevcube) [inline modules/aurora-postgres/variables.tf:1]: all removed variables were unused.

File: modules/aurora-postgres/ssm.tf [hunk 1]
```diff
@@ -67,14 +65,6 @@ locals {
...
 }
 
-data "aws_ssm_parameter" "password" {
-  count = local.fetch_admin_password ? 1 : 0
-
-  name = format(var.ssm_password_source, local.admin_user)
-
-  with_decryption = true
-}
-
```
[the remark this answers is not in the document]
Comment (kevcube) [reply modules/aurora-postgres/ssm.tf:77]: removing this makes the local unused, and also some variables.

[the remark this answers is not in the document]
Comment (kevcube) [reply modules/aurora-postgres/ssm.tf:77]: if the intention was to use this, i.e with remote state, then it should be stored in a sensitive output.

**Review by milldr — DISMISSED** [review]
LGTM, thanks for finding these unused inputs

**Review by milldr — APPROVED** [review]

**Pull request merged**